Presentation.Web/app/components/org/structure/org-structure-modal-edit.view.html

```diff
@@ -1,64 +1,81 @@
 ﻿<div data-ng-if="!createNew">
-    <div class="modal-header">
+    <div class="modal-header wide-modal">
         <h3>Rediger {{::orgUnit.oldName}}</h3>
         <span class="help-block small" ng-if="supplementaryText !== null">{{::supplementaryText}}</span>
     </div>
-    <div class="modal-body">
-        <h4>Rediger info</h4>
+    <div class="modal-body wide-modal">
         <form name="editForm" novalidate="novalidate" class="form-horizontal">
-            <div class="form-group">
-                <label for="parent-select" class="col-xs-3 control-label">
-                    Overordnet enhed
-                </label>
-                <div class="col-xs-8">
-                    <single-select-select2 id="parent-select" ng-model="parentSelect" ng-disabled="!canChangeParent" />
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <span>Info</span>
                 </div>
-                <div class="col-xs-8 col-xs-offset-3">
-                    <p class="help-block" ng-if="canChangeParent">
-                        Der kan kun vælges blandt de organisationsenheder som er indenfor samme organisation, og som ikke er en underenhed til {{::orgUnit.oldName}}.
-                    </p>
-                    <p class="help-block" ng-if="!canChangeParent">
-                        Du kan ikke ændre {{ ::orgUnit.oldName + 's' }} overordnet organisationsenhed.
-                        <span data-ng-show="!isAdmin">Kontakt en lokal administrator.</span>
-                    </p>
+                <div class="panel-body">
+                    <div class="form-group">
+                        <label for="parent-select" class="col-xs-3 control-label">
+                            Overordnet enhed
+                        </label>
+                        <div class="col-xs-8">
+                            <single-select-select2 id="parent-select" ng-model="parentSelect" ng-disabled="!canChangeParent" />
+                        </div>
+                        <div class="col-xs-8 col-xs-offset-3">
+                            <p class="help-block" ng-if="canChangeParent">
+                                Der kan kun vælges blandt de organisationsenheder som er indenfor samme organisation, og som ikke er en underenhed til {{::orgUnit.oldName}}.
+                            </p>
+                            <p class="help-block" ng-if="!canChangeParent">
+                                Du kan ikke ændre {{ ::orgUnit.oldName + 's' }} overordnet organisationsenhed.
```

Comment:
change text to 

`Du kan ikke ændre overordnet organisationsenhed for "{{ ::orgUnit.oldName}}"`